test/system/src/test/java/io/pravega/test/system/MultiSegmentStoreTest.java

```diff
@@ -127,21 +155,33 @@ public void testMultiSegmentStores() throws InterruptedException {
         testReadWrite();
 
         // Scale up and test.
-        this.segmentServiceInstance.scaleService(3, true);
+        try {
+            Exceptions.handleInterrupted(() -> this.segmentServiceInstance.scaleService(3).get());
+        } catch (ExecutionException e) {
+            throw new TestFrameworkException(RequestFailed, "Scaling operation failed", e);
+        }
         // Wait for all containers to be up and registered.
         Thread.sleep(60000);
         log.info("Test with 3 segment stores running");
         testReadWrite();
 
         // Rescale and verify.
-        this.segmentServiceInstance.scaleService(2, true);
+        try {
+            Exceptions.handleInterrupted(() -> this.segmentServiceInstance.scaleService(2).get());
+        } catch (ExecutionException e) {
+            throw new TestFrameworkException(RequestFailed, "Scaling operation failed", e);
+        }
         // Wait for all containers to be up and registered.
         Thread.sleep(60000);
         log.info("Test with 2 segment stores running");
         testReadWrite();
 
         // Rescale to single instance and verify.
-        this.segmentServiceInstance.scaleService(1, true);
+        try {
+            Exceptions.handleInterrupted(() -> this.segmentServiceInstance.scaleService(1).get());
+        } catch (ExecutionException e) {
```

Comment:
Here and above Futures.getAndHandleExceptions().

Reply:
done.